docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,427 @@
+.. _admin:
+
+=====
+Admin
+=====
+
+This is the cog guide for the admin cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load admin
+
+.. _admin-usage:
+
+-----
+Usage
+-----
+
+This cog will provide tools for server admins and bot owners.
+
+It can add or remove a role to a member, edit one or make some available
+for members so they can self-assign them as they wish.
+
+It also provides tools for the bot owner such as server locking (once enabled,
+the bot will instantly leave new servers she joins) and announcements, which
```

Comment:
Nope, `she` is refering to red.

Reply:
Yes. Red is female, and is referred to as such throughout all guides, because the character her name is based on (Red from Transistor) is female.

Also replacing with "he" would have the same "problem" so that isn't a solution.

Reply:
Even if it isn't he, wouldn't the correct term be "it" here since we're not talking about the character red but a piece of software?

Reply:
Originally it was he in the docs, but it was then changed to she in my first batch of review stuffs.

Reply:
That's actually wanted, and we have discussed about it before. Red is a girl, as a reference to the original character from Transistor. (see https://github.com/Cog-Creators/Red-DiscordBot/pull/2659#discussion_r331033748)